pkg/apis/servicecatalog/types.go

```diff
@@ -266,10 +266,16 @@ type InstanceSpec struct {
 	// provisioned from.
 	PlanName string
 
-	// Parameters is a YAML representation of the properties to be
+	// Parameters is set of properties to be
 	// passed to the underlying broker.
 	Parameters *runtime.RawExtension
 
+	// Optional: AlphaSecretParameters is reference to the secret object containing
```

Comment:
Can you explain in the godoc for `Parameters` and `AlphaSecretParameters` that only one of these fields can be set at a time?  Also, explain the behavior how the secret is translated into the parameter payload.